src/common/types/core.js

```diff
@@ -406,6 +406,12 @@ type Reply = {
     type: 'reply',
 };
 
+type DOMStringList = {
```

Comment:
Maybe add a comment as to why we need to define this type ourselves

Reply:
I've added a comment